GitUI/UserControls/FileStatusList.cs

```diff
@@ -1534,13 +1537,25 @@ private void FileStatusList_Enter(object sender, EventArgs e)
         private readonly Subject<string> _filterSubject = new Subject<string>();
         [CanBeNull] private Regex _filter;
         private bool _filterVisible = false;
+        private bool _filterActive = false;
```

Comment:
Why is this used?
OnFilterChanged() may fire the event more than once, is that so bad?
(these status variablles are hard to maintain, someon else reuses it for a slightly separate use case or add a parallel variable...)

Reply:
It fires on every character added/removed. But you're right, it would be no problem here without it so I can remove that check. Or maybe just rename it to something more meaningful like _filterActiveLastState or _filterActiveCheck? (and sorry for my delayed answer)

Reply:
Just to be sure - Which one do you prefer?
- remove the unnecessary check? or
- rename to something more meaningful?

Reply:
_I_ prefer maintainability, so removing the variable if there is no noticeable effect is what I prefer.